docs/data-sources/cluster.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
I previously deleted fatherboat documentation, as not to clutter it with unmaintained items